gammapy/spectrum/tests/test_extract.py

```diff
@@ -105,8 +105,8 @@ def test_sherpa(self, tmpdir, extraction):
         desired = extraction.observations[0].aeff.data.data.value
         assert_allclose(actual, desired)
 
-    def test_define_energy_threshold(self, extraction):
+    def test_compute_energy_threshold(self, extraction):
         # TODO: Find better API for this
```

Comment:
I think this TODO is resolved now, isn't it?

Reply:
I guess? Do you want me to remove it?

Reply:
Yes, please :wink: